app/models/annotation/handler/AnnotationInformationHandler.scala

```diff
@@ -22,6 +23,8 @@ class AnnotationInformationHandlerSelector @Inject()(projectInformationHandler:
 
 trait AnnotationInformationHandler extends FoxImplicits {
 
+  implicit val ec: ExecutionContext
```

Comment:
this abstract val is implemented in the implementing classes (by there prefixing the injected ec with `val`), In this way we make the ExecutionContext available also in the trait’s methods